Stockchecker.js

```diff
@@ -84,4 +83,4 @@ function doesntexist(element, name) {
 
 
 
-console.log(doesntexist(product, 'spoon'));
\ No newline at end of file
+console.log(doesntexist(product, 'Table'));
```

Comment:
instock function should not do any console.logging. it should return a new array with only products that are in stock